vitest.config.mts

```diff
@@ -17,7 +17,6 @@ function generateConfig(): Promise<UserConfigExport> {
             '**/themes/**',
             '**/.{idea,git,cache,output,temp}/**',
             '**/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build}.config.*',
-            'arches_references/constants.ts',
```

Comment:
This was hastily added in a prior PR but in another repo (lingo?) we've avoided making this change, so just bringing back in sync.